### ISSUE TITLE
including speaking url in file name

### ISSUE THREAD

**Issue by alexanderadam** [gautamdhameja/medium-2-md#4]
It would be nice if filenames would be converted into the 'speaking' urls instead article identifier.

So, for example if I want to archive an article with
```bash
$ medium-2-md convertUrl https://medium.com/@watzon/doing-crystal-2-501daaf8390d -o . -fi
```

it is resulting in `501daaf8390d.md`. By looking at it's filename I'm not able to understand anymore what the article was about.

Maybe something like `watzon_-_doing-crystal-2-501daaf8390d.md` would be more speaking and recognizable?

What do you think @gautamdhameja?

**Comment by gautamdhameja** [comment]
I think this makes sense. I will look into it. Thanks.

**Comment by gautamdhameja** [comment]
Fixed in v0.4.1. https://www.npmjs.com/package/medium-2-md/v/0.4.1

**Comment by alexanderadam** [comment]
Awesome, thank you!
